packages/chain-bridge/src/adapters/LayerZeroAdapter.sol

```diff
@@ -131,21 +138,23 @@ contract LayerZeroAdapter is Ownable, OAppRead, IBridgeAdapter {
      * @notice Sets the minimum gas limit for a specific message type
      * @param msgType Message type to set minimum gas for
      * @param gasLimit New minimum gas limit value
-     * @dev Can only be called by the contract owner
+     * @dev Can be called by super keeper for operational tuning
      */
     function setMinGasLimit(
         uint16 msgType,
         uint128 gasLimit
-    ) external onlyOwner {
+    ) external onlySuperKeeper {
         minGasLimits[msgType] = gasLimit;
     }
 
     /**
      * @notice Activates a read channel for state reading operations
      * @param _readChannelId The ID of the read channel to activate
-     * @dev Can only be called by the contract owner
+     * @dev Long-term configuration - deployer during deployment, governance after transition
      */
-    function activateReadChannel(uint32 _readChannelId) external onlyOwner {
+    function activateReadChannel(
```

Comment:
wouldn't it be better to seed the initial data set in constructor during deployment ? seems like all the data is known before deployment, the read channel, the supported chains and eIDs. Bridge router can be added to the upcoming registry

Reply:
I'm not questioning the option to add / remove supported chains - I'm questioning the `deployment phase` in general, as all the params could be passed to the constructor, without overly complicating/extending the access. Especially that the removal would be handeled during the post deployment phase - so in either case it would be the governance performing the removal.